### PR TITLE
Weather Metrics App Development

### PR DESCRIPTION
**Pull request by Epaltechs**
### In this project, I implement the following:
- Use the React-Redux template to set up the project.
- Create the Home page so that when the page loads, the webapp shows the list of categories that are filtered out.
- Along with the category name, display some numeric values per country that come from the API.
- When the user clicks (or taps) on a country, the application navigates to the details page.
- On the details page, the webapp retrieves data from the API to show detailed data in the country category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- Create unit tests for pure functions in the Redux code.
- Create integration tests for the application using the React Testing Library.
- Deployed the project online.
- Description of the project in a demo showing the project features.
- Give credit to the design author in the README file of the project.

### CODE REVIEW

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Epaltechs , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
Your Linters are not configured properly. Please make sure the Eslint config is present in the linters.yml file. Check out this [link ](https://github.com/microverseinc/linters-config/tree/master/react-redux)and try to follow the directions there.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
\ No newline at end of file
+# Weather-Metrics-App
+
+This metrics weather application uses the weather updates fetched from [Open Weather API](https://openweathermap.org/api) to display weather updates of different capital cities in the continent of Africa.
+
+## DEMO
+
+<img src="./src/assets/Weather Demo.gif" alt="app-demo"/>
+
+[Deployment](https://epaltechs.github.io/weather-metrics-app/)
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Kindly deploy the app using heroku or netlify and provide a live link.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Epaltechs, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR description and title looks professional ✔️
- Descriptive commit messages ✔️
- No linter errors 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Use arrow functions (ES6) for all the methods.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Epaltechs** [comment]
### In this commit, I deployed the project on Netlify and Heroku and add the link to the README.md file as required by the code review.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Epaltechs 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done making the requested changes
- Nicely done deploying your app
- Good communication

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**